app/controllers/games_controller.rb

```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
+
+# Comment necessary for Rubocop
+class GamesController < ApplicationController
+  before_action :set_game, only: %i[show update destroy]
+
+  # GET /games
+  def index
+    @games = Game.all
+
+    render json: @games
+  end
+
+  # GET /games/1
+  def show
+    render json: @game
+  end
+
+  # POST /games
+  def create
+    Game.delete_all
+    new_game = TicTacToeLogic::GameBoardData.new
+    @game = Game.new({ "game_state": "IN-PROGRESS", "current_player": new_game.current_player.to_s, "board": new_game })
+
+    if @game.save
+      render json: @game, status: :created, location: @game
+    else
+      render json: @game.errors, status: :unprocessable_entity
+    end
+  end
+
+  def submit_move
+    @game = Game.find(1)
+    @game.board.play_move(params[:move].to_i)
+    @game.current_player = @game.board.current_player
+    @game.game_state = @game.board.game_over? ? "GAME OVER" : "IN-PROGRESS"
```

Comment:
it feels weird that we are calculating and storing these values, we should not need to store this and instead should be calculating these values when we render the json.

Reply:
So will I not need a database at all?

Reply:
You should store the game board [:player_one, :player_two ...] . But you should not be saving the game_state and current_player in the database since we can calculate those when we need them. Also, we run the risk of having the game_state or current player in the database being incorrect for the board.